.github/workflows/linux_cuda.yml

```diff
@@ -0,0 +1,204 @@
+        uses: actions/checkout@v3
+        with:
+          repository: 'openvinotoolkit/openvino_contrib'
+          path: 'openvino_contrib'
+          submodules: 'recursive'
+
+      - name: Clone testdata for C API tests
+        uses: actions/checkout@v3
+        with:
+          repository: 'openvinotoolkit/testdata'
+          path: 'testdata'
+          submodules: 'recursive'
+          lfs: 'true'
+
+      - name: Create Directories
+        run: |
+          mkdir -p $GITHUB_WORKSPACE/build
+          mkdir -p $GITHUB_WORKSPACE/install_pkg
+
+      - name: Setup Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      #
+      # Dependencies
+      #
+
+      - name: Install build dependencies
+        run: |
+          apt update
+          $GITHUB_WORKSPACE/openvino/install_build_dependencies.sh
+          
+          apt -y --no-install-recommends install unzip wget software-properties-common
+          wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
+          unzip ninja-linux.zip
+          cp -v ninja /usr/local/bin/
+          
+          python3 -m pip install -r $GITHUB_WORKSPACE/openvino/src/bindings/python/requirements.txt
+
+      - name: Install CUDA
+        run: |
+         wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin 
+         mv cuda-ubuntu2004.pin /etc/apt/preferences.d/cuda-repository-pin-600
+         
+         apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/3bf863cc.pub
+         add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/ /"
+         apt update
+         DEBIAN_FRONTEND=noninteractive apt install -y \
+            gnupg2 \
+            software-properties-common \
+            libcudnn8=8.9.4.*-1+cuda11.8 \
+            libcudnn8-dev=8.9.4.*-1+cuda11.8 \
+            libcudnn8-samples=8.9.4.*-1+cuda11.8 \
+            autoconf \
+            automake \
+            build-essential \
+            ca-certificates \
+            clang-format-9 \
+            cuda-runtime-11-8 \
+            cuda-11-8 \
+            g++-multilib \
+            gcc-multilib \
+            gstreamer1.0-plugins-base \
+            libavcodec-dev \
+            libavformat-dev \
+            libboost-regex-dev \
+            libcairo2-dev \
+            libcutensor1=1.6.1.5-1 \
+            libcutensor-dev=1.6.1.5-1 \
+            libglib2.0-dev \
+            libgstreamer1.0-0 \
+            libgtk2.0-dev \
+            libopenblas-dev \
+            libpango1.0-dev \
+            libpng-dev \
+            libssl-dev \
+            libswscale-dev \
+            libtool \
+            libusb-1.0-0-dev \
+            libzstd-dev \
+            pkg-config \
+            shellcheck \
+            unzip \
+            wget \
+            cuda-drivers=520.61.05-1
```

Comment:
let's install only CUDA packages, all others are not required and covered by install_build_dependencies.sh